tests/integration/devfile/cmd_devfile_url_test.go

```diff
@@ -53,7 +54,7 @@ var _ = Describe("odo devfile url command tests", func() {
 			helper.CmdShouldPass("git", "clone", "https://github.com/che-samples/web-nodejs-sample.git", projectDirPath)
 			helper.Chdir(projectDirPath)
 
-			helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, componentName)
+			helper.CmdShouldPass("odo", "create", "nodejs", componentName, "--project", namespace)
```

Comment:
This change is nothing to deal with the flake fix. I just put the component name adjacent to componentType to make whole command more sync with our UX examples.